templates/service-template.yml

```diff
@@ -539,6 +539,24 @@ parameters:
   description: Special Management Organization ID used for creating internal Service accounts
   value: "13640203"
 
+- name: STRIMZI_OPERATOR_SUBSCRIPTION_CONFIG
+  displayName: Strimzi operator subscription config
+  description: Strimzi operator subscription config. This is applied for standalone clusters only. The configuration must be of type https://pkg.go.dev/github.com/operator-framework/api@v0.3.25/pkg/operators/v1alpha1?utm_source=gopls#SubscriptionConfig
+  value: "{}"
+
+- name: KAS_FLEETSHARD_OPERATOR_SUBSCRIPTION_CONFIG
+  displayName: Kas-fleetshard operator subscription config
+  description: kas-fleetshard operator subscription config. This is applied for standalone clusters only. The configuration must be of type https://pkg.go.dev/github.com/operator-framework/api@v0.3.25/pkg/operators/v1alpha1?utm_source=gopls#SubscriptionConfig
+  value: "{}"
```

Comment:
@MikeEdgar the 
- `STRIMZI_OPERATOR_SUBSCRIPTION_CONFIG` and
- `KAS_FLEETSHARD_OPERATOR_SUBSCRIPTION_CONFIG`

parameters are what can be used to configure the subscription spec config for the two operators.

Reply:
Thank you - I'll pull the latest and try it out now

Reply:
Thank you.